docs/pages/docs/editor-basics/setup.mdx

```diff
@@ -33,6 +33,16 @@ type BlockNoteEditorOptions = {
     class?: string;
   }) => Plugin;
   initialContent?: PartialBlock[];
+  pasteHandler?: (context: {
+    view: EditorView;
+    event: ClipboardEvent;
+    editor: BlockNoteEditor;
+    defaultPasteHandler: (context: {
+      pasteBehavior?: "prefer-markdown" | "prefer-html";
+    }) => boolean | undefined;
+    convertHtmlToBlockNoteHtml: (html: string) => string;
```

Comment:
API feedback:

- what about exposing `pasteHTML` `pasteMarkdown` and `pasteText` on the editor instead of the conversion functions? this way consumers also don't need to call into prosemirror `view.paste...` functions.
- I'm not sure I like passing the `defaultPasteHandler` vs exporting that function directly (still in doubt)

Reply:
- I like exposing the methods, to keep it simple for users, I exposed pasteHTML, pasteText & pasteMarkdown methods
 - I think having too many things exposed on a top-level API make it hard for discoverability, this function would only be useful for pasting content, so it makes sense to expose on the handler you'd use it at.